dataset_configs/armenian/audio_books.yaml

```diff
@@ -0,0 +1,59 @@
+processors_to_run: "0:"
+workspace_dir: ???
+
+processors:
+  - _target_: sdp.processors.CreateInitialManifestByExt
+    raw_data_dir: ${workspace_dir}/mp3
+    extension: mp3
+    output_field: source_filepath
+    output_manifest_file: ${workspace_dir}/manifest0.json
+
+  - _target_: sdp.processors.FfmpegConvert
+    output_manifest_file: ${workspace_dir}/manifest1.json
+    resampled_audio_dir: ${workspace_dir}/audio
+    target_samplerate: 16000
+    target_nchannels: 1
+    input_field: "source_filepath"
+    output_field: "audio_filepath"
+    key_field: null
+
+  - _target_: sdp.processors.GetAudioDuration
+    audio_filepath_field: audio_filepath
+    duration_field: duration
+    output_manifest_file: ${workspace_dir}/manifest2.json
+    
+  - _target_: sdp.processors.ASRWhisper
+    output_manifest_file: ${workspace_dir}/manifest3.json
+    pretrained_model: "large-v2"
+    output_text_field: text
+
+  - _target_: sdp.processors.SubMakeLowercase
+    output_manifest_file: ${workspace_dir}/manifest4.json
```

Comment:
For me, output_manifest_file helps to remember the number of the step N to use in processors_to_run: "N:". Otherwise, I need to count this number from the beginning each time

Reply:
Another option could be to add a naming of steps and call them in 'processors_to_run' by unique names.